### ISSUE TITLE
deprecated dependency: request@2.88.2

### ISSUE THREAD

**Issue by petebacondarwin** [angular/angular-cli#16952]
It appears that this version, which is deprecated is being pulled in by `@angular/cli` (via `universal-analytics`). I assume a bump to the latter's version will be enough to fix this?

You can see this warning when doing a fresh install of CLI@9.0.1:

```
warning @angular/cli > universal-analytics > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
```

**Comment by lequanglam** [comment]
there is no latter version of request.

**Comment by wartab** [comment]
The paackage `request` is now fully deprecated. So that will probably take a while to change.

**Comment by petebacondarwin** [comment]
Yes, I think the solution would be for `universal-analytics` to release a new version that is not reliant upon `request`.

**Comment by AURZeeshan** [comment]
problem !     request@2.88.2..


what are the alternative of it ?

**Comment by BruneXX** [comment]
Hi, any update on this? thanks!

**Comment by prog98rammer** [comment]
@BruneXX 
Me too, I'm waiting for a fix or something

**Comment by BruneXX** [comment]
I think we'll need to check here: https://github.com/request/request/issues/3142

**Comment by smoncadav** [comment]
I'm also getting this issue when installing angular CLI

`npm install -g @angular/cli`
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

**Comment by gerardodinardo** [comment]
¿and how I install Angular now? I was able to install it via WSL, but not on Windows.

**Comment by akhilh2o** [comment]
I'm also waiting for this solution===>

npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
C:\Users\hp\AppData\Roaming\npm\ng -> C:\Users\hp\AppData\Roaming\npm\node_modules\@angular\cli\bin\ng

> @angular/cli@9.1.1 postinstall C:\Users\hp\AppData\Roaming\npm\node_modules\@angular\cli
> node ./bin/postinstall/script.js

? Would you like to share anonymous usage data with the Angular Team at Google under
Google’s Privacy Policy at https://policies.google.com/privacy? For more details and
how to change this setting, see http://angular.io/analytics. No
+ @angular/cli@9.1.1
added 272 packages from 207 contributors in 341.271s

**Comment by Serine18** [comment]
i have the same probleme,has any one found a solution 

npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
C:\Users\pc pro\AppData\Roaming\npm\ng -> C:\Users\pc pro\AppData\Roaming\npm\node_modules\@angular\cli\bin\ng

**Comment by alan-agius4** [comment]
This is currently blocked as it requires an upstream change.

https://github.com/peaksandpies/universal-analytics/issues/143

**Comment by alan-agius4** [comment]
This is currently blocked as it requires an upstream change.

https://github.com/peaksandpies/universal-analytics/issues/143

**Comment by fToledo29** [comment]
Any updates about this issue?

**Comment by Serine18** [comment]
Hi,for me i could create a angular project so i ingore this warn and every thing work correctly. تم الإرسال من البريد لـ Windows 10 من: Fernando Toledo Ramirezإرسال: 22 avril 2020 21:02إلى: angular/angular-cliنسخة: Serine bkb; Commentالموضوع: Re: [angular/angular-cli] deprecated dependency: request@2.88.2 (#16952) Any updates about this issue?—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by jburson** [comment]
Looks like their only usage is https://github.com/peaksandpies/universal-analytics/blob/master/lib/index.js#L444

**Comment by beeman** [comment]
I submitted [a PR](https://github.com/peaksandpies/universal-analytics/pull/144) to `peaksandpies/universal-analytics` but the last commit to that project was 1.5 year ago so I doubt it will land anytime soon. Also, while upgrading said project, I got blocked on one of the unit tests (see PR).

**Comment by alan-agius4** [comment]
Closed via #18113

**Comment by alan-agius4** [comment]
Reopening since this was reverted in https://github.com/angular/angular-cli/pull/18173
